google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -179,6 +206,99 @@ public Builder setSessionLabels(Map<String, String> sessionLabels) {
       return this;
     }
 
+    /**
+     * {@link SpannerOptions.Builder} does not support global retry settings, as it creates three
+     * different gRPC clients: {@link Spanner}, {@link DatabaseAdminClient} and {@link
+     * InstanceAdminClient}. Instead of calling this method, you should set specific {@link
+     * RetrySettings} for each of the underlying gRPC clients by calling respectively {@link
+     * #getSpannerStubSettingsBuilder()}, {@link #getDatabaseAdminStubSettingsBuilder()} or {@link
+     * #getInstanceAdminStubSettingsBuilder()}.
+     */
+    @Override
+    public Builder setRetrySettings(RetrySettings retrySettings) {
+      throw new UnsupportedOperationException(
+          "SpannerOptions does not support setting global retry settings. "
+              + "Call spannerStubSettingsBuilder().<method-name>Settings().setRetrySettings(RetrySettings) instead.");
+    }
+
+    /**
+     * Returns the {@link SpannerStubSettings.Builder} that will be used to build the {@link
+     * SpannerRpc}. Use this to set custom {@link RetrySettings} for individual gRPC methods.
+     *
+     * <p>The library will automatically use sensible defaults if no custom settings are set. The
```

Comment:
small comment - can we replace "sensible defaults" to a link to where the defaults are set?

Reply:
I've added a reference to the defaults for all three.